Services/Mail/PRIVACY.md

```diff
@@ -0,0 +1,80 @@
+# Mail Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The `Mail Service` can be disabled in the global administration or restricted to user accounts assigned
+  to certain roles.
+- The delivery to external email addresses via `SMTP` or `Sendmail` can be enabled and configured in the
+  global administration. External email delivery (concrete: using explicit email addresses as recipient) can be
```

Comment:
like abvove please state what kind of account can do this with what permission at what click path